tools/NodeConfigCompare/configuration/cloud_formation_template.yaml

```diff
@@ -281,6 +282,57 @@ Resources:
         ServerSideEncryptionConfiguration:
           - ServerSideEncryptionByDefault:
               SSEAlgorithm: AES256
+      LoggingConfiguration:
+        DestinationBucketName: !Ref LoggingBucket
+        LogFilePrefix: NodeConfigCompare
+  
+  S3BucketPolicyConfigTestingBucket:
+    Type: 'AWS::S3::BucketPolicy'
+    Properties: 
+      Bucket: !Ref RedshiftConfigTestingBucket
+      PolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Action:
+            - 's3:PutObject'
+            - 's3:GetObject'
+            - 's3:ListBucket'
+            Effect: Allow
+            Resource:
+              - !Join ['',['arn:aws:s3:::',!Ref RedshiftConfigTestingBucket]]
+              - !Join ['',['arn:aws:s3:::',!Ref RedshiftConfigTestingBucket, '/*']]
+            Principal:
+              Service:
+                - lambda.amazonaws.com
+                - batch.amazonaws.com
+                - redshift.amazonaws.com
+                - states.amazonaws.com
+            
+  LoggingBucket:
+    Type: 'AWS::S3::Bucket'
+    Properties:
+      BucketEncryption:
+        ServerSideEncryptionConfiguration:
+          - ServerSideEncryptionByDefault:
+              SSEAlgorithm: AES256
+
+  S3BucketPolicy:
+    Type: 'AWS::S3::BucketPolicy'
+    Properties:
+      Bucket: !Ref LoggingBucket
+      PolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Action:
+              - 's3:PutObject'
+            Effect: Allow
+            Principal:
+              Service: logging.s3.amazonaws.com
+            Resource: !Join 
+              - ''
+              - - 'arn:aws:s3:::'
+                - !Ref LoggingBucket
+                - /*
```

Comment:
I would use the same convention you have used in the other bucket policy.

Reply:
👍